docs/src/modules/java/pages/component-and-service-calls.adoc

```diff
@@ -37,9 +37,7 @@ of endpoints the service provides.
 === Over HTTP
 The service is identified by the name it has been deployed. Akka takes care of routing requests to the service and keeping the data safe by encrypting the connection and handling authentication for you.
 
-In this sample we will make an action that does a call to the xref:key-value-entities.adoc[Key Value Entity Counter] service, deployed with the service name `counter`.
-
-The SDK provides `akka.javasdk.http.HttpClientProvider` which provides HTTP client instances for calling other services.
+In the follow snippet, we have an endpoint component that calls another service named `"counter"`. It makes use of SDK-provided `akka.javasdk.http.HttpClientProvider` which return HTTP client instances for calling other Akka services.
```

Comment:
I don't see any benefit or need to explain that the counter service has an endpoint and a KV entity behind it. Clearer this way I think.